### PR TITLE
Transfer some demos to Enterprise

### PR DESCRIPTION
**Pull request by FlorianJacta**
Part of https://github.com/Avaiga/taipy-enterprise/issues/399

[Gallery - Taipy.zip](https://github.com/user-attachments/files/15985597/Gallery.-.Taipy.zip)

image

image

image

### CODE REVIEW

**Review by jrobinAV — APPROVED** [review]
Looks good to me!

Thanks for the screenshots. They make the PR easy to review.

File: docs/gallery/decision_support/3_production_planning/index.md [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
 img: 3_production_planning/images/production-planning-data-visualization.png
 ---
-Production planning is pivotal in manufacturing, optimizing resources to meet
-demand while minimizing costs. This demo offers a solution for efficient production and cost management.
+Optimizing resources to meet demand while minimizing costs is pivotal in manufacturing. This is why 
+production planning tools are mandatory in manufacturing.
 
-[Try it live](https://production-planning.taipy.cloud/Data-Visualization){: .tp-btn target='blank' }
-[Get it on GitHub](https://github.com/Avaiga/demo-production-planning){: .tp-btn .tp-btn--accent target='blank' }
+!!! note "Enterprise application"
```
Comment (FabienLelaquais) [inline docs/gallery/decision_support/3_production_planning/index.md]: I don't like this title.
Why not "Taipy Enterprise edition"?
Funny you don't even refer to that in the text body...

File: docs/gallery/decision_support/3_production_planning/index.md [hunk 2]
```diff
@@ -20,21 +30,16 @@ In this demo, an optimization algorithm, based on the PuLP open-source math solv
...
 
-
 ![Data Visualization](images/production-planning-data-visualization.png){width=90% : .tp-image-border }
 
-Just above the chart, by clicking "Expand here," you can access an expandable
-Taipy front-end containing initial production data at time 0 (current month):
-stock & production levels, incoming raw material orders, and demand, all presented in a table.
-
-
+Just above the chart, by clicking "Expand here," you can access an expandable containing initial
```
Comment (FabienLelaquais) [inline docs/gallery/decision_support/3_production_planning/index.md]:
```suggestion
Just above the chart, by clicking "Expand here," you can access an expandable block containing the initial 
```

File: docs/gallery/decision_support/3_qsr_sales_forecasting/index.md [hunk 3]
```diff
@@ -11,7 +11,15 @@ business problem involves time series forecasting for several core business acti
 the QSR, such as sales made on drinks or the number of transactions made at the front 
 counter.
 
-[Try it live](https://qsr-fcst.taipy.cloud/data){: .tp-btn target='blank' }
+!!! note "Enterprise application"
```
Comment (FabienLelaquais) [inline docs/gallery/decision_support/3_qsr_sales_forecasting/index.md]: Same remark

File: docs/gallery/decision_support/bar_cutting/index.md [hunk 4]
```diff
@@ -10,7 +10,15 @@ to minimize waste. Bar-cutting optimization minimizes waste
 by choosing optimal cutting patterns. This is crucial in industries
 with significant material costs such as Steel, Timber, etc.
 
-[Try it live](https://bar-cutting.taipy.cloud/Databases){: .tp-btn target='blank' }
+!!! note "Enterprise application"
```
Comment (FabienLelaquais) [inline docs/gallery/decision_support/bar_cutting/index.md]: Same remark

File: docs/gallery/decision_support/churn_classification/index.md [hunk 5]
```diff
@@ -11,8 +11,16 @@ more cost-effective than acquiring new ones. By identifying
 customers who may leave early and applying retention strategies,
 businesses can lower churn rates and boost customer loyalty.
 
-[Try it live](https://churn-classification.taipy.cloud/Data-Visualization){: .tp-btn target='blank' }
-[Get it on GitHub](https://github.com/Avaiga/demo-churn-classification){: .tp-btn .tp-btn--accent target='blank' }
+!!! note "Enterprise application"
```
Comment (FabienLelaquais) [inline docs/gallery/decision_support/churn_classification/index.md]: Same remark

File: docs/gallery/finance/fraud_detection/index.md [hunk 6]
```diff
@@ -1,14 +1,22 @@
...
-data-keywords: dashboard vizelement layout chart ai multi-page classification
+data-keywords: dashboard vizelement layout chart ai multi-page classification enterprise
 short-description: A Taipy Application that analyzes credit card transactions to detect fraud.
 img: fraud_detection/images/fraud_threshold.png
 ---
 A Taipy Application that analyzes credit card transactions to detect fraud.
 
-[Try it live](https://fraud-detection.taipy.cloud/Transactions?null=){: .tp-btn target='blank' }
-[Get it on GitHub](https://github.com/Avaiga/demo-fraud-detection){: .tp-btn .tp-btn--accent target='blank' }
+!!! note "Enterprise application"
```
Comment (FabienLelaquais) [inline docs/gallery/finance/fraud_detection/index.md]: Same remark

File: docs/gallery/llm/6_talk_to_taipy/index.md [hunk 7]
```diff
@@ -1,13 +1,21 @@
...
 category: llm
-data-keywords: dashboard vizelement layout chart ai
+data-keywords: dashboard vizelement layout chart ai enterprise
 short-description: Explore datasets using only natural language using TalkToTaipy!
 img: 6_talk_to_taipy/images/talk_to_taipy_image.jpeg
 ---
 Explore datasets using only natural language using TalkToTaipy!
 
-[Try it live](https://talk-to-taipy.taipy.cloud/){: .tp-btn target='blank' }
+!!! note "Enterprise application"
```
Comment (FabienLelaquais) [inline docs/gallery/llm/6_talk_to_taipy/index.md]: Same remark

File: docs/gallery/other/index.md_template [hunk 8]
```diff
@@ -37,6 +37,18 @@ See other applications using Taipy.
       <span>Optimization</span>
     </label>
   </li>
+    <li>
+    <input type="checkbox" name="filter-community" id="filter-community" value="community">
+    <label class="tp-pill" for="filter-community">
+      <span>Community</span>
```
Comment (FabienLelaquais) [inline docs/gallery/other/index.md_template]:
```suggestion
      <span>Community edition</span>
```

**Review by AlexandreSajus — APPROVED** [review]
Great job!

File: docs/gallery/other/index.md_template [hunk 9]
```diff
@@ -37,6 +37,18 @@ See other applications using Taipy.
...
+    </label>
+  </li>
+  <li>
+    <input type="checkbox" name="filter-enterprise" id="filter-enterprise" value="enterprise">
+    <label class="tp-pill" for="filter-enterprise">
+      <span>Enterprise</span>
```
Comment (FabienLelaquais) [inline docs/gallery/other/index.md_template]:
```suggestion
      <span>Enterprise edition</span>
```

File: docs/gallery/llm/index.md_template [hunk 10]
```diff
@@ -37,6 +37,18 @@ Uncover a list of applications using LLM.
       <span>Optimization</span>
     </label>
   </li>
+  <li>
+    <input type="checkbox" name="filter-community" id="filter-community" value="community">
+    <label class="tp-pill" for="filter-community">
+      <span>Community</span>
```
Comment (FabienLelaquais) [inline docs/gallery/llm/index.md_template]:
```suggestion
      <span>Community edition</span>
```

File: docs/gallery/llm/index.md_template [hunk 11]
```diff
@@ -37,6 +37,18 @@ Uncover a list of applications using LLM.
...
+    </label>
+  </li>
+  <li>
+    <input type="checkbox" name="filter-enterprise" id="filter-enterprise" value="enterprise">
+    <label class="tp-pill" for="filter-enterprise">
+      <span>Enterprise</span>
```
Comment (FabienLelaquais) [inline docs/gallery/llm/index.md_template]:
```suggestion
      <span>Enterprise edition</span>
```

File: docs/gallery/index.md_template [hunk 12]
```diff
@@ -110,6 +110,18 @@ Let's explore applications made with Taipy.
       <span>Optimization</span>
     </label>
   </li>
+  <li>
+    <input type="checkbox" name="filter-community" id="filter-community" value="community">
+    <label class="tp-pill" for="filter-community">
+      <span>Community</span>
```
Comment (FabienLelaquais) [inline docs/gallery/index.md_template]:
```suggestion
      <span>Community edition</span>
```

File: docs/gallery/index.md_template [hunk 13]
```diff
@@ -110,6 +110,18 @@ Let's explore applications made with Taipy.
...
+    </label>
+  </li>
+  <li>
+    <input type="checkbox" name="filter-enterprise" id="filter-enterprise" value="enterprise">
+    <label class="tp-pill" for="filter-enterprise">
+      <span>Enterprise</span>
```
Comment (FabienLelaquais) [inline docs/gallery/index.md_template]:
```suggestion
      <span>Enterprise edition</span>
```

File: docs/gallery/finance/index.md_template [hunk 14]
```diff
@@ -37,6 +37,18 @@ Explore financial applications using Taipy.
       <span>Optimization</span>
     </label>
   </li>
+  <li>
+    <input type="checkbox" name="filter-community" id="filter-community" value="community">
+    <label class="tp-pill" for="filter-community">
+      <span>Community</span>
```
Comment (FabienLelaquais) [inline docs/gallery/finance/index.md_template]:
```suggestion
      <span>Community edition</span>
```

File: docs/gallery/finance/index.md_template [hunk 15]
```diff
@@ -37,6 +37,18 @@ Explore financial applications using Taipy.
...
+    </label>
+  </li>
+  <li>
+    <input type="checkbox" name="filter-enterprise" id="filter-enterprise" value="enterprise">
+    <label class="tp-pill" for="filter-enterprise">
+      <span>Enterprise</span>
```
Comment (FabienLelaquais) [inline docs/gallery/finance/index.md_template]:
```suggestion
      <span>Enterprise edition</span>
```

File: docs/gallery/decision_support/index.md_template [hunk 16]
```diff
@@ -37,6 +37,18 @@ Discover decision support applications for what-if analysis.
       <span>Optimization</span>
     </label>
   </li>
+  <li>
+    <input type="checkbox" name="filter-community" id="filter-community" value="community">
+    <label class="tp-pill" for="filter-community">
+      <span>Community</span>
```
Comment (FabienLelaquais) [inline docs/gallery/decision_support/index.md_template]:
```suggestion
      <span>Community edition</span>
```

File: docs/gallery/decision_support/index.md_template [hunk 17]
```diff
@@ -37,6 +37,18 @@ Discover decision support applications for what-if analysis.
...
+    </label>
+  </li>
+  <li>
+    <input type="checkbox" name="filter-enterprise" id="filter-enterprise" value="enterprise">
+    <label class="tp-pill" for="filter-enterprise">
+      <span>Enterprise</span>
```
Comment (FabienLelaquais) [inline docs/gallery/decision_support/index.md_template]:
```suggestion
      <span>Enterprise edition</span>
```

File: docs/gallery/visualization/index.md_template [hunk 18]
```diff
@@ -37,6 +37,18 @@ Delve into applications that focus on visualization.
       <span>Optimization</span>
     </label>
   </li>
+    <li>
+    <input type="checkbox" name="filter-community" id="filter-community" value="community">
+    <label class="tp-pill" for="filter-community">
+      <span>Community</span>
```
Comment (FabienLelaquais) [inline docs/gallery/visualization/index.md_template]:
```suggestion
      <span>Community edition</span>
```

File: docs/gallery/visualization/index.md_template [hunk 19]
```diff
@@ -37,6 +37,18 @@ Delve into applications that focus on visualization.
...
+    </label>
+  </li>
+  <li>
+    <input type="checkbox" name="filter-enterprise" id="filter-enterprise" value="enterprise">
+    <label class="tp-pill" for="filter-enterprise">
+      <span>Enterprise</span>
```
Comment (FabienLelaquais) [inline docs/gallery/visualization/index.md_template]:
```suggestion
      <span>Enterprise edition</span>
```

**Review by FabienLelaquais — CHANGES_REQUESTED** [review]
Not sure why you never refer to Taipy Enterprise edition

**Review by FabienLelaquais — APPROVED** [review]
👌

**Pull request merged**